builds/Makefile

```diff
@@ -390,9 +390,7 @@ TOOLS64_CONFIGURE_FLAGS=	--build=x86_64-apple-darwin10 \
 			--cache-file=../tools64.config.cache \
 			--with-glib=embedded \
 			--enable-minimal=com	\
-			--with-profile2=no \
 			--with-profile4=no \
```

Comment:
This still doesn't switch off the 4.x profile in Mono, if that's what was intended here?